docs/news.d/general.rst

```diff
@@ -0,0 +1,37 @@
+.. news-prs: 4298 4292 4262 4295 4226 4293 4253 4252 4250 4265 4254 4219 4216
+
+.. news-start-section: Additions
+- Support sending DynamicDataAdapter sample via DynamicDataWriter
+.. news-end-section
+
+.. news-start-section: Fixes
+- Improvements
```

Comment:
Is this a relevant place to use subsections (https://opendds.readthedocs.io/en/master/internal/docs.html#docs-news-subsections)? Or maybe this does the same thing?

Reply:
Subsections are similar to manually creating a list, but subsections can be shared by fragments, like they share "Additions", "Fixes", etc., while a RST list can not be shared. After separating this fragment into multiple fragments, I think you could use subsections if they had common topics, but I'm not sure these really do.